### ISSUE TITLE
Accessibility Issue: aria-command-name

### ISSUE THREAD

**Issue by wegnertm** [amcharts/amcharts4#4379]
When running the [aXe **](https://www.deque.com/axe/) I'm getting `aria-command-name` errors on my [pie chart series](https://brightstart.com/investments/index-balanced/) and on my [bar chart series](https://brightstart.com/cost-of-college/). 

The issue is that the series is focusable but doesn't have an accessible name. Upon further investigation it looks like an aria-label can be added if I add `series.itemReaderText` in my code but this only appears once I've tabbed to a slice. Could this aria-label be automatically added to the HTML output instead of just showing on focus? Then it would clear the scan. 

 - amCharts4
 - Custom Wordpress Theme
 - PHP

Thanks!

**Comment by martynasma** [comment]
I'm afraid amCharts 4 is on feature freeze with only critical issues being fixed.

This does not seems to be critical. In addition, requiring changing attribute application as you suggest would require a bit of an overhaul for accessibility functionality, which we can't afford for an EOL product.

I hope this sounds reasonable.
